rln/src/utils.rs

```diff
@@ -121,7 +121,7 @@ pub fn vec_u8_to_bytes_be(input: Vec<u8>) -> Vec<u8> {
 pub fn bytes_le_to_vec_u8(input: &[u8]) -> (Vec<u8>, usize) {
     let mut read: usize = 0;
 
-    let len = usize::try_from(u64::from_le_bytes(input[0..8].try_into().unwrap())).unwrap();
+    let len = u64::from_le_bytes(input[0..8].try_into().unwrap()) as usize;
```

Comment:
This conversion should be safe, especially if we change lengths from u64 to u32. (See https://github.com/vacp2p/zerokit/issues/39)